frontend/src/metabase/new_query/containers/NewQueryOptions.jsx

```diff
@@ -61,6 +61,7 @@ export class NewQueryOptions extends Component {
                 description={t`Pick some data, view it, and easily filter, summarize, and visualize it.`}
                 width={180}
                 to={Urls.newQuestion()}
+                data-metabase-event={`New Question; Simple Question Start`}
```

Comment:
Also is there a reason to use template strings instead of just `data-metabase-event="..."`?

Reply:
imo we shouldn't do that , since it's not like we want these internationalized?